### DISCUSSION TITLE
What database does STATS__BLOCKSCOUT_DB_URL refer to?

### DISCUSSION THREAD

**Discussion by XO39** [blockscout/blockscout#10012] [Q&A]
We are trying to run the microservices  directly without a docker as we can't use docker for some reason.
We are are confused about the database that the STATS__BLOCKSCOUT_DB_URL env variable refers to.

Is it the same database that the DATABASE_URL refers to? Or it is a completely different database?

**Comment by Sadeghi1017** [comment]
https://bscscan.com/tx/0xaf7e7c169ef06f911dfc24b1dc7a52e10d2bf9eb3545f123d4a983bb49df91d6#eventlog?locale=en-US&utm_source=imtoken

**Comment by Sadeghi1017** [comment]
> https://bscscan.com/tx/0xaf7e7c169ef06f911dfc24b1dc7a52e10d2bf9eb3545f123d4a983bb49df91d6#eventlog?locale=en-US&utm_source=imtoken

**Comment by sl1depengwyn** [comment]
> Is it the same database that the DATABASE_URL refers to?

Yes, it is
